elements/timeline-view/scrolling-box.html

```diff
@@ -72,16 +76,27 @@
                     duration: 0,
                     easing: "linear",
                 });
+
+            const drawer = document.getElementsByTagName("paper-drawer-panel");
+            let containerEl = drawer[0];
```

Comment:
Please write full variable names. We had trouble in the past reading variables in your code.